src/device.coffee

```diff
@@ -40,8 +39,16 @@ exports.getID = memoizePromise ->
 			throw new Error('Could not find this device?!')
 		return devices[0].id
 
+waitForReboot = ->
+	console.log('Waiting for reboot...')
+	Promise.delay(3000).then(waitForReboot)
```

Comment:
We've had cases where a device tries to reboot and then never does, meaning this will just spam our device logs like crazy - it also doesn't stop the supervisor doing other stuff (which I'm guessing was the aim?)